### DISCUSSION TITLE
Unable to write SNES SRM files onto cart coming from Emulator.

### DISCUSSION THREAD

**Discussion by b3nner** [sanni/cartreader#985] [Need help]
I am having issues with writing snes srm save files back onto snes carts when coming from an emulator. I am constantly getting verification errors (Error:1312 bytes did not verify). 

Has anyone else encountered this issue? I've tried using https://savefileconverter.com/ but nothing seems to work. 

I do a lot of RPG grinding on emulator using fast forward and I would love to get these save files back onto my carts.

Using the Sanni Cart Reader V5

any help would be appreciated. thanks.

**Comment by Chomemel** [comment]
I had the exact same problem with a French translation of Bahamut Lagoon. Turns out it came form the ROM and more precisely the patch. There was an error on the speed of the game. A friend of mine corrected it and everything went find after that.
What game gave you the error?

**Comment by sanni** [comment]
If you dump the save from the cartridge does it have the same file size as the emulators save you are trying to write?

**Comment by b3nner** [comment]
problem has been resolved thanks to the help of a few people on the oscr discord channel, it turns out I had to reflow some of the snes solder pins on the cartridge header. thank you.
